### ISSUE TITLE
Is it possible to release a new version of xsalsa20poly1305 now that zeroize is not blocked to <1.4?

### ISSUE THREAD

**Issue by fabuloso** [RustCrypto/AEADs#422]
We are encountering `zeroize` version conflicts between `xsalsa20poly1305` and `aws-sdk`. 

```
error: failed to select a version for `zeroize`.
    ... required by package `aws-types v0.10.1`
    ... which satisfies dependency `aws-types = "^0.10.1"` of package `aws-config v0.10.1`
    ... which satisfies dependency `aws-config = "^0.10"` of package `my_project v0.1.0 (...)`
versions that meet the requirements `^1.4.1` are: 1.5.4, 1.5.5, 1.5.3, 1.4.3, 1.4.2, 1.4.1

all possible versions conflict with previously selected packages.

  previously selected package `zeroize v1.0.0`
    ... which satisfies dependency `zeroize = ">=1, <1.4"` of package `salsa20 v0.9.0`
    ... which satisfies dependency `salsa20 = "^0.9"` of package `xsalsa20poly1305 v0.8.0`
    ... which satisfies dependency `xsalsa20poly1305 = "^0.8.0"` of package `my_project v0.1.0 (...)`

failed to select a version for `zeroize` which could resolve this conflict
```

Could you please release a new version? We cannot upgrade aws-sdk at the moment, and dependabot is really angry. 😢

**Comment by tarcieri** [comment]
I've released `xsalsa20poly1305` v0.9.0-pre which unpins `zeroize`: #423

We'll need to get a new version of `universal-hash` out before a final release. See https://github.com/RustCrypto/traits/pull/965

**Comment by fabuloso** [comment]
thanks a lot
